eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -37,9 +37,9 @@
           }
       },
       "TestMode": {
-          "netcoreapp21_Coverage_Record": {
+          "netcoreapp21_Coverage_or_Record": {
               "SupportsRecording": true,
-              "CollectCoverage": true,
+              "CollectCoverage": false,
```

Comment:
Why not opt-out? Projects should care about coverage (eventually managers will want a report on it, I was told) and have it just work by default (on).